src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5585,6 +5587,28 @@ CompareTasksByTaskId(const void *leftElement, const void *rightElement)
 }
 
 
+/*
+ * CompareTasksByExecutionDuration is a helper function to compare two tasks by their execution duration.
+ */
+int
+CompareTasksByExecutionDuration(const void *leftElement, const void *rightElement)
```

Comment:
this method could possibly be in `multi_explain.c` as a static method as it is only used there

Reply:
I thought that it would be nice if I implement this next to `CompareTasksByTaskId`, since they are each other's alternatives. What do you think? Should I move this to `multi_explain.c` anyway?

Reply:
Yes, let's move the multi_explain.c. Note that we don't have `taskExecutionTime` for all tasks, so we should  even rename the function to `CompareTasksByFetchedExplainAnalyzeDuration`